workflow-ui/core-android/src/main/java/com/squareup/workflow1/ui/container/DialogSession.kt

```diff
@@ -46,14 +49,17 @@ internal class DialogSession(
    * Wrap the given dialog holder to maintain [allowEvents] on each update.
    */
   val holder: OverlayDialogHolder<Overlay> = OverlayDialogHolder(
-    holder.environment, holder.dialog
+    holder.environment, holder.dialog, holder.onUpdateBounds, holder.onBackPressed
```

Comment:
For `onUpdateBounds` we weren't routing through the `onUpdateBounds` of the original holder before? So it was re-creating the default argument?

Reply:
Yes, big oops.